### ISSUE TITLE
Toolbox icons broken in input-rich-text (WYSIWYG)

### ISSUE THREAD

**Issue by sguter90** [directus/directus#18208]
### Describe the Bug

It seems that since the release of https://github.com/directus/directus/pull/18004 the icons for bold, italic and underline aren't displayed correctly.
<img width="834" alt="grafik" src="https://user-images.githubusercontent.com/4067165/232334790-1806fd4c-75dc-4da2-b188-cfc454ac7a4a.png">

As a workaround I added the following css rules as a dirty hack to custom CSS:
```css
.tox-icon::after {
    display: none !important;
}

.tox-tbtn .tox-icon svg {
    display: inline-block !important;
}
```

As far I can see in the component, there is no configured icon setting for tinymce:
https://github.com/directus/directus/blob/main/app/src/interfaces/input-rich-text-html/input-rich-text-html.vue

Maybe the import of ``tinymce/icons/default`` conflicts with some other import or the icon logic does not match anymore.

### To Reproduce

1. Create data model with WYSIWYG field
2. Open "Create" form for data model
3. The first three toolbar icons show text instead of the icon

### Directus Version

v9.25.1

### Hosting Strategy

Self-Hosted (Docker Image)

**Comment by grefrit** [comment]
Fixed by #18185

**Comment by alsherif-khalaf** [comment]
Same problem after upgrade
